fipy/tools/dimensions/physicalField.py

```diff
@@ -1622,8 +1622,8 @@ def __rtruediv__(self, other):
         if isinstance(other,PhysicalUnit):
             return PhysicalUnit(other.names-self.names,
                                 other.factor/self.factor,
-                                map(lambda a,b: a-b,
-                                    other.powers, self.powers))
+                                list(map(lambda a,b: a-b,
```

Comment:
This seems weird. Most of the other changes in this commit involve changing a map to a generator, which makes sense. I don't think explicitly casting a `map` to a `list` is the same thing. I wonder why these aren't just generators, too.